src/components/BottomBarCourse.vue

```diff
@@ -162,6 +175,12 @@ export default {
     width: 100%;
     background-color: #FFF;
 
+    &-cancel{
+      float:right;
+      padding: 5px 25px 0px 10px;
```

Comment:
We should use rem instead of px values for spacing for accessibility purposes in case the user's default font size is changed. Can you convert these pixels to REM instead? (And use factors of 8 when possible).